model/stripe/model.sql

```diff
@@ -0,0 +1,132 @@
+create or replace view {{env.schema}}.stripe_invoices as (
+
+  select
+    customer,
+    date,
+    forgiven,
+    subscription as subscription_id,
+    paid,
+    total,
+    period_start,
+    period_end
+  from
+    demo_data.stripe_invoices
+
+);
+
+create or replace view {{env.schema}}.stripe_invoices_cleaned as (
+
+  select
+    customer,
+    timestamp 'epoch' + date * interval '1 Second' as date,
+    forgiven,
+    subscription_id,
+    paid,
+    total,
+    timestamp 'epoch' + period_start * interval '1 Second' as period_start,
+    timestamp 'epoch' + period_end * interval '1 Second' as period_end
+  from
+    {{env.schema}}.stripe_invoices
+
+);
+
+create or replace view {{env.schema}}.stripe_subscriptions as (
+
+  select
+    id,
+    plan__id as plan_id
+  from
+    demo_data.stripe_subscriptions
+
+);
+
+create or replace view {{env.schema}}.stripe_plans as (
+
+  select
+    id,
+    interval
+  from
+    demo_data.stripe_plans
+
+);
+
+
+
+
+create or replace view {{env.schema}}.stripe_invoices_transformed as (
+
+  with invoices as (
+
+    select *
+    from {{env.schema}}.stripe_invoices_cleaned
+    where paid is true
+      and forgiven is false
+
+  ), d1 as (
+
+    select (min(period_start) over () + row_number() over ())::date as date_day
+    from invoices
+
+  ), dates as (
```

Comment:
@jthandy `months` instead of `dates`, if limiting to only months in this table. We may want to add `date_day` and `date_quarter` since these will be other common time periods over which we may want to look.